administrator-manual/en/webtop5.rst

```diff
@@ -249,6 +248,38 @@ the check box is named :guilabel:`Show quick preview on message row`.
 
 .. image:: _static/webtop-preview.png
 
+Mail archiving
+==============
+
+Archiving is useful for keeping your inbox folder organized by moving manually the messages.
+
+.. note::
+    Mail archiving is not a backup.
+
+The system automatically creates a new special Archives folder  
+
+.. image:: _static/webtop-archive_archive1.png
+
+If the :guilabel:`Archives` folder does not appear immediately upon login, it will appear at the first archiving.
+
+ There are three archiving criteria in :menuselection:`Settings -> Mail -> Archiving`
+
+* **Single folder:** a single root for all archived emails
+* **Per year:** a root for each year
+* **By year / month:** a root for each year and month
+
+.. image:: _static/webtop-archive_archive2.png
+
+To maintain the original structure of the folders is possible to activate **"Keep folder structure"** 
```

Comment:
Enclose "Keep folder structure" inside the ``guilabel`` tag.

Reply:
done